src/System.Numerics.Vectors/ref/System.Numerics.Vectors.csproj

```diff
@@ -16,12 +16,9 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.0-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Release|AnyCPU'" />
-  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
```

Comment:
We may want to also include this in the package since NS.L no longer has the forwarder.  Its not a strict requirement since we already have a S.R-based ref that will apply to NS2.0.  Including this one in the package would merely add a copy that uses ns.dll as the core assembly instead of S.R.